RELEASE.md

```diff
@@ -0,0 +1,22 @@
+# Releasing the scorecard GitHub Action
+
+This is a draft document to describe the release process for the scorecard
+GitHub Action.
+
+(If there are improvements you'd like to see, please comment on the
+[tracking issue](https://github.com/ossf/scorecard-action/issues/33) or issue a
+pull request to discuss.)
+
+- [steps:](#steps)
+
+## steps:
+
+1. cut a scorecard release and wait for a container image to be created and tagged with new release. Note the hash of the container as `CH1`. Note: we do not need a scorecard release, we can use any stable version we want.
```

Comment:
none of these hashes are used in scripts.

CH1 is the Container image's Hash for the scorecard container (scorecard repo). There's one container generated for every stable build (after it's gone thru integration tests) and for each release. We use this container as part of a multi-stage build to pull the scorecard binary in https://github.com/ossf/scorecard-action/blob/main/Dockerfile#L34

HG2 means Github Hash: it's the commit hash of this repo (scorecard-action) after we've updated the docker image to use the scorecard container image we want.

Reply:
Thanks Laurent.
I just wanted to make sure!